.github/workflows/test.yaml

```diff
@@ -0,0 +1,73 @@
+name: Build docker images
+
+concurrency:
+  group: test-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
```

Comment:
disable `pull_request:` trigger in base.yaml?

Reply:
Unsure about that. I mean we'd still want to run some some checks if some of the base Dockerfiles have changed? Though I'm also unsure that running `docker/build-push-action` with `pull: true` (as it is now) would give us the feedback we need.

Reply:
May be remove raspbian, we don't use it for building packages as of now.